src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -1,49 +1,50 @@
 package com.wikia.webdriver.elements.common;
 
-import com.wikia.webdriver.common.contentpatterns.URLsContent;
-import org.joda.time.DateTime;
-
 import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
 
 public class Navigate extends BasePageObject {
 
   public Navigate toPage(String pageName) {
-    String query = getQueryParams(pageName);
-
-    driver.get(urlBuilder.getUrlForWiki() + URLsContent.WIKI_DIR + pageName + query);
+    driver.get(urlBuilder.getUrlForWikiPage(pageName) + urlBuilder.getCacheBusterQuery(pageName));
 
     return this;
   }
 
   public Navigate toPageByPath(String path) {
-    String query = getQueryParams(path);
+    String query = urlBuilder.getCacheBusterQuery(path);
+
+    driver.get(urlBuilder.getUrl() + path + query);
 
-    driver.get(urlBuilder.getUrlForWiki() + path + query);
+    try {
+      Thread.sleep(2000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)